quic/core/frames/quic_mc_client_limits_frame.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "quic/core/frames/quic_mc_client_limits_frame.h"
+
+namespace quic {
+
+QuicMcClientLimitsFrame::QuicMcClientLimitsFrame(
+    QuicControlFrameId control_frame_id,
+    QuicClientLimitsSequenceNumber client_limits_sn,
+    QuicMcClientLimitsCapabilitiesField capabilities,
```

Comment:
We should probably change the semantics on the transport property too though right? They should probably be the same

Reply:
Probably consistent is better.  Is there nothing else in quic that does this kind of thing outside the frame types?

I guess not really.  The closest seems to be the least significant bits of the stream frame or the datagram frame types, which are a little like this, I think, but otherwise they seem to have mostly kept this kind of thing out of the protocol?

Preferred address is another example that does this kind of "could be IPv4 or IPv6" thing but that just ran with "always both, you can use a special address to say 'not this one'".   Now that I'm looking at it, it seems a deliberate and odd design choice, and I wonder what drove it...